tests/run-tests.sh

```diff
@@ -430,14 +430,12 @@ then
         --map-by ppr:8:node \
         -x LD_LIBRARY_PATH=/usr/local/nccl-rdma-sharp-plugins/lib:$LD_LIBRARY_PATH \
         -mca coll_hcoll_enable 0 \
-        -x NCCL_IB_PCI_RELAXED_ORDERING=1 \
         -x UCX_IB_PCI_RELAXED_ORDERING=on \
```

Comment:
we could remove this. but lets not block this PR, we can do it in a follow up PR.